runelite-client/src/main/java/net/runelite/client/game/chatbox/ChatboxTextMenuInput.java

```diff
@@ -56,7 +56,7 @@ private static final class Entry
 	private String title;
 
 	@Getter
-	private List<Entry> options = new ArrayList<>();
+	private final List<Entry> options = new ArrayList<>();
```

Comment:
are all of these types of changes (setting final on getter/setter fields) then also unnecessary? Ideally I'd like the minimum amount of change that keeps pmd happy.

Reply:
Is the ImmutableFieldRule flagging it? Since it really looks like it shouldn't https://github.com/pmd/pmd/commit/66ddd5617f0c7ea71ed71e13b083b0555b6ad0a0#diff-f0d84f4b43ab1c1e2fbc8b8ee09c4c0c3ee2c5a7b981d99fae749652821ec07aR64